### ISSUE TITLE
Dimensions starting with LookML keywords break parsing

### ISSUE THREAD

**Issue by ryantuck** [joshtemple/lkml#11]
Dimensions prefaced with `html` or `sql` throw parsing errors, likely because they are valid expected keywords.

My guess is that similar issues would happen with other keywords. I'd expect this not to happen.

If I get a chance to dig around in the code to see how to fix it, I'll open a PR.

```
$ pip list | grep lkml
lkml                          0.1.1
$ cat lkml_test.view.lkml
view: blah {
  sql_table_name: test ;;

  # these prefixes break parsing
  dimension: html_blah {}
  dimension: sql_stuff {}
}
$ lkml lkml_test.view.lkml
Traceback (most recent call last):
  File "/usr/local/bin/lkml", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.7/site-packages/lkml/__init__.py", line 59, in cli
    lookml = load(args.file)
  File "/usr/local/lib/python3.7/site-packages/lkml/__init__.py", line 13, in load
    tokens = lexer.scan()
  File "/usr/local/lib/python3.7/site-packages/lkml/lexer.py", line 81, in scan
    self.tokens.append(self.scan_expression_block())
  File "/usr/local/lib/python3.7/site-packages/lkml/lexer.py", line 92, in scan_expression_block
    if self.peek() == "\n":
  File "/usr/local/lib/python3.7/site-packages/lkml/lexer.py", line 13, in peek
    return self.text[self.index]
IndexError: string index out of range
```

**Comment by joshtemple** [comment]
This is an issue with the [Lexer](https://github.com/joshtemple/lkml/blob/master/lkml/lexer.py), which goes through the text and attempts to break it up into tokens. If it encounters `sql`, `html`, or `expression_custom_filter` it tries to parse an [`ExpressionBlockToken`](https://github.com/joshtemple/lkml/blob/master/lkml/tokens.py#L57-L65) (ending with `;;`). In this case, that's the wrong decision since it's not an expression block token, it's a [`LiteralToken`](https://github.com/joshtemple/lkml/blob/master/lkml/tokens.py#L68-L76). Here's the [logic for that decision](https://github.com/joshtemple/lkml/blob/master/lkml/lexer.py#L71-L75).

To fix this, `lkml` will probably need to be more specific about which strings trigger the start of an expression block. For instance, that could be represented as nested logic checking for the actual strings, e.g. `sql_table_name`.

Feel free to submit a PR, otherwise I'll target this fix in `0.1.2`.

**Comment by dwmintz** [comment]
Ran into this as well, but looks like y'all are already on it!
